releasenotes/notes/gradients-primitives-561cf9cf75a7ccb8.yaml

```
@@ -0,0 +1,14 @@
+---
+features:
+  - |
+    New gradient classes using the primitives has been added. They internally
```

```suggestion
    New gradient algorithms using the primitives have been added. They internally
```
Since we called the fidelities algorithms in the release note I suggest algorithms here too,